manifests/r/RobloxCorporation/RobloxBootstrapper/Latest/RobloxCorporation.RobloxBootstrapper.yaml

```diff
@@ -0,0 +1,8 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.4.0.schema.json
+
+PackageIdentifier: RobloxCorporation.RobloxBootstrapper
+PackageVersion: Latest

```

Comment:
Latest is not a valid package version. Please use a semantic version, even if it is arbitrary

Reply:
apply change